### ISSUE TITLE
Unable to trigger mutations with externalEventMutations

### ISSUE THREAD

**Issue by alexgyllos** [FormidableLabs/victory-native#613]
TL;DR: I can't figure out how to make externalEventMutations target a specific slice of the pie chart and reduce its innerRadius.

I am working with victory-native as I am developing for mobile. I am trying to go through the data array I have by letting the user press on any part of the pie wrapper within the TouchableOpacity.

Currently, I am able to update the label that is displayed in the middle of the pie to reflect on each element of the data array. However, I would also like to reduce the appropriate slice's innerRadius to reflect on which slice the label is representing.

I tried the example on the docs but couldn't make it work.

My function shuffleBusinessPie does the following:

 const shuffleBusinessPie = () => {
    if (businessLabel.length === 0) {
      return;
    }
    if (businessCounter < businessLabel.length) {
      setBusinessCounter(businessCounter + 1);
      setBusinessPieLabel([businessLabel[businessCounter][0], businessLabel[businessCounter][1]]);
      setBusinessExternalMutations([
        {
          childName: `business-pie`,
          target: ["data"],
          eventKey: "all",
          mutation: () => ({ style: { fill: "#fff" } }),
        },
      ]);
      console.log(externalMutations);
    } else {
      setBusinessPieLabel([businessLabel[0][0], businessLabel[0][1]]);
      setBusinessCounter(1);
    }
  };

Currently working with Victory Pie like so:

 <TouchableOpacity onPress={() => shuffleBusinessPie()}>
                <VictoryPie
                  name="business-pie"
                  padAngle={5}
                  innerRadius={80}
                  cornerRadius={2}
                  colorScale={["#3870c3", "#4d536b", "#faaf3b", "#FF5722", "#d8dade"]}
                  labelRadius={20}
                  labelComponent={
                    <VictoryLabel
                      textAnchor="middle"
                      verticalAnchor="middle"
                      style={[
                        {
                          fontSize: 25,
                          color: "#28323B",
                          fill: "white",
                          fontWeight: "600",
                          lineHeight: 40,
                        },
                        { fontWeight: "600", fontSize: 20, fill: "white" },
                      ]}
                      x={200 * 0.5}
                      y={200 * 0.525}
                      text={[businessPieLabel[0], businessPieLabel[1]]}
                    />
                  }
                  style={{
                    data: {
                      fill: "grey",
                    },
                  }}
                  height={props.height || 200}
                  width={props.width || 200}
                  padding={0}
                  data={businessMilesDict}
                  externalEventMutations={externalMutations}
                />
              </TouchableOpacity
